roles/user-interface/tasks/main.yml

```diff
@@ -1 +1,58 @@
 ---
+- name: Install login_checks.sh script.
```

Comment:
Makes sense to also move the yum install of the pam-script RPM to this role.

Reply:
Good point.